### PR TITLE
FISH-5703 Option to Improve Logging Speed

### PR DESCRIPTION
**Pull request by JamesHillyard**
## Description
A new feature allowing users to skip the forced .toString() on all parameters when logging. The command to set this property has been added to set-log-attributes and the admin console. This property is updated dynamically and doesn't need a server restart for the logging changes to take affect.

### Blockers
None

### New tests
None

### Testing Performed
Manually tested the toString is skipped appropriately and without the need for a server restart. Ran the logging tests already in place.

### Testing Environment
Windows 10, JDK 8, Maven 3.6.3

## Documentation
Community Documentation PR: https://github.com/payara/Payara-Community-Documentation/pull/276

## Notes for Reviewers
If you test this manually, fastLoggingEnabled() will return the old value while logging a logging property has been changed. Any logs after this will be using the new value. This doesn't impact the functionality but please keep it in mind if you manually test.

### CODE REVIEW

**Comment by JamesHillyard** [comment]
Jenkins test please

File: nucleus/common/common-util/src/main/java/com/sun/common/util/logging/GFLogRecord.java [hunk 1]
```diff
@@ -135,4 +144,19 @@ public static GFLogRecord wrap(LogRecord record, boolean setThreadName) {
         }
         return result;
     }
+
+    /**
+     * Reads the logging.properties file for fastLogging property dynamically so server restart is not needed
+     * @return true if fast logging is enabled
+     */
+    private static boolean fastLoggingEnabled() {
```
Comment (rdebusscher) [inline nucleus/common/common-util/src/main/java/com/sun/common/util/logging/GFLogRecord.java:152]: This is slow and defeats the purpose of making logging faster.

**Review by rdebusscher — CHANGES_REQUESTED** [review]

**Comment by JamesHillyard** [comment]
Jenkins test please

**Review by rdebusscher — COMMENTED** [review]
Looks good

**Comment by JamesHillyard** [comment]
Jenkins test please

**Review by aubi — APPROVED** [review]

**Review by rdebusscher — APPROVED** [review]

**Pull request merged**